grammar.js

```diff
@@ -614,7 +626,9 @@ module.exports = grammar({
 
     elvis_expression: $ => prec.left(PREC.ELVIS, seq($._expression, "?:", $._expression)),
 
-    check_expression: $ => prec.left(PREC.CHECK, seq($._expression, choice($._in_operator, $._is_operator), $._expression)),
+    check_expression: $ => prec.left(PREC.CHECK, seq($._expression, choice(
+									   seq($._in_operator, $._expression),
+									   seq($._is_operator, $._type)))),
```

Comment:
@colleend that's the other fix to is_operator I mentioned in your PR to fwcd.

Reply:
oh awesome, I'll also include this one then!